### PR TITLE
Make scalafixRules publishable

### PR DESCRIPTION
**Pull request by xeno-by**

### CODE REVIEW

**Comment by rscbenchbox** [comment]
Accepted benchmarking [9f53785](https://github.com/xeno-by/rsc/commit/9f5378513b750d0c971534af983a02bbb5b7ab76). This may take a while.

File: project/Build.scala [hunk 1]
```diff
@@ -45,10 +45,35 @@ object Build extends AutoPlugin {
     shellout(bin +: args, cwd)
   }
 
-  implicit class FileOps(file: File) {
+  private implicit class FileOps(file: File) {
     def abs: String = file.getAbsolutePath
```
Comment (dwijnand) [inline project/Build.scala]: (complete side-note: this is also available as [absolutePath](https://github.com/sbt/sbt/blob/v0.13.17/util/io/src/main/scala/sbt/Path.scala#L28) if you'd like)

Comment (xeno-by) [reply project/Build.scala]: Very nice!! I'm going to use this right away. Thank you :)

**Comment by rscbenchbox** [comment]
Accepted benchmarking [1d68a64](https://github.com/xeno-by/rsc/commit/1d68a643e114096cecba6a069052cad1f8c31689). This may take a while.

**Comment by rscbenchbox** [comment]
Finished benchmarking [1d68a64](https://github.com/xeno-by/rsc/commit/1d68a643e114096cecba6a069052cad1f8c31689).

| Benchmark     | Before | After |
|---------------|--------|-------|
| RscParse      | 19.793 ms     | 18.865 ms    |
| RscLink       | 52.856 ms     | 54.797 ms    |
| RscOutline    | 38.933 ms     | 39.577 ms    |
| RscSemanticdb | 27.761 ms     | 23.984 ms    |
| RscMjar       | 291.906 ms     | 267.555 ms    |
| ScalacCompile | ~9x     | ~10x    |

Full results are available at https://github.com/twitter/rsc/commit/40dbf7133c7a01bcf8aab64aab4408549e218086.

**Pull request merged**